tutteli-gradle-junitjacoco/src/main/groovy/ch/tutteli/gradle/junitjacoco/JunitJacocoPluginExtension.groovy

```diff
@@ -2,25 +2,23 @@ package ch.tutteli.gradle.junitjacoco
 
 import org.gradle.api.Action
 import org.gradle.api.Project
-import org.gradle.api.Task
 import org.gradle.api.provider.Property
 import org.gradle.testing.jacoco.plugins.JacocoPluginExtension
 import org.gradle.testing.jacoco.tasks.JacocoReport
-import org.junit.platform.gradle.plugin.JUnitPlatformExtension
 
 class JunitJacocoPluginExtension {
-    private Task junitPlatformTestTask
     private JacocoPluginExtension jacocoPluginExtension
     private JacocoReport jacocoReportTask
-    private JUnitPlatformExtension junitPlatformExtension
 
+    /**
+     * @deprecated Use the gradle API instead: test { reports { junitXml { enabled = false }}}
+     */
+    @Deprecated(/*since = "0.28.0"*/)
```

Comment:
I like the comment since when it is deprecated :slightly_smiling_face:

Reply:
actually, why do you use a comment and not `@Deprecated(since = "0.28.0")`?

Reply:
Because the `since` parameter was only introduced in Java 9 (see [Javadoc for version 8](https://docs.oracle.com/javase/8/docs/api/)).

Reply:
Good